source/common/json/config_schemas.cc

```diff
@@ -1205,6 +1205,10 @@ const std::string Json::Schema::CLUSTER_SCHEMA(R"EOF(
         "enum" : ["http2"]
       },
       "http_codec_options" : {"type" : "string"},
+      "dns_resolvers": {
+        "type" : "array",
```

Comment:
It is a valid configuration, but a weird one - it would be the same as if you just didn't have "dns_resolvers" present at all. Should we enforce there to be at least 1 element?